pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -1184,6 +1195,18 @@ func (b *MasqueradeBindMechanism) createNatRulesUsingNftables(proto iptables.Pro
 		return err
 	}
 
+	for _, lmPort := range portsUsedByLiveMigration() {
+		for _, chain := range []string{"output", "KUBEVIRT_POSTINBOUND"} {
```

Comment:
There's a couple of things that might "complicate" the code a little bit, but would help the reader a lot; please consider them:

```golang
const (
    chainWhereDnatIsPerformed = "output"
    chainWhereSnatIsPerformed = "KUBEVIRT_POSTINBOUND"
)
...
chainsToAvoidForReservedPorts := []string{chainWhereDnatIsPerformed, chainWhereSnatIsPerformed}
...
```

Not sure if you agree with me.

I would even go a bit further, and encapsulate this whole function in an helper, whose name would help the reader understand what your'e doing. How about `skipNatForReservedPorts(...)` ?

Reply:
I agree with wrapping this in a helper method.

I'm not sure about the constants though -`output` is a chain where various actions can be performed, not necessarily DNAT.

Reply:
> I agree with wrapping this in a helper method.
> 
> I'm not sure about the constants though -`output` is a chain where various actions can be performed, not necessarily DNAT.

Yes, of course - but my motivation is for you to make it easy for whoever reads this to understanding why we're bailing out of that particular chain that, as you point out, does plenty of things.

For your scenario, you want to avoid DNATing on that table, SNATing on the other. 

This is just a suggestion; while I'm trying to help future me understand what happens there, I can't promise future me will have an easier time. I think so though.